plutus-core/cost-model/create-cost-model/CostModelCreation.hs

```diff
@@ -75,6 +74,7 @@ costModelsR = do
     source("cost-model/data/models.R")
     modelFun("cost-model/data/benching.csv")
   |]
+  -- Unfortunately we can't use the paths defined in FilePaths inside [r|...].  The above code may not work on Windows.
```

Comment:
... but we only need to run this code ourselves, to calibrate our cost model on a Linux reference machine.  Is shouldn't matter if it doesn't work on Windows.

Reply:
OK, so it sounds as if this wasn't actually what was causing the problem.  I'm not sure if this will actually make any difference to the behaviour on Windows, but it's probably still worth using proper path operations rather than having /-separated paths included in the code.

Reply:
Yeah, maybe add a comment that we normally only run this on linux.

Reply:
OK, why not?